### ISSUE TITLE
Option to disable icon auto-arrange on desktop is assigned to the wrong monitor when using two monitors (primary and secondary are switched)

### ISSUE THREAD

**Issue by mendel5** [linuxmint/mint20-beta#148]
I have been using Linux Mint 19.3 Cinnamon and Linux Mint 20 Cinnamon Beta on the same computer. This computer has two monitors a Nvidia graphics card. In the settings it is necessary to set one of the two monitors as the primary one. That works so far.

On my desktop I like to arrange the icons/ launchers in a custom order. Therefore I disable the "Auto-arrange" option for the primary monitor in the setting "Current Monitor Layout".

On Linux Mint 19.3 Cinnamon it works like this:
* Right-click on an empty space on the desktop
* The context menu opens up
* Click on "Customize"
* Here I can disable "Auto-arrange"
* I do this process on the primary monitor where the icons are supposed to be. When I try to do this on the secondary monitor it says "Desktop icons are not currently enabled for this monitor.". Everything works as expected.

However when I do the same on the Linux Mint 20 Cinnamon Beta it's different. When I click "Customize" in the context menu on the primary monitor it says "Desktop icons are not currently enabled for this monitor.". Then I click "Customize" on the secondary monitor and can change the auto-arrange settings for the primary monitor.

So the problem is:
* When clicking on the Customize option in the context menu on the **primary monitor** I can adjust the settings of the **secondary monitor**.
* When clicking on the Customize option in the context menu on the **secondary monitor** I can adjust the settings of the **primary monitor**.

**Comment by mendel5** [comment]
I experienced this bug yesterday and only created this issue now. In the meantime the bug does not occur anymore. I am not sure what fixed it, but here is a list of some things I have changed:
* Update Manager: Install all updates
* Driver Manager: Switch from Nvidia nouveau open source driver to Nvidia closed source driver
* Welcome Screen: Switch from standard panel layout to "Traditional" panel layout (see also https://github.com/linuxmint/mint20-beta/issues/147 ).

I am not sure if one of these changes actually fixed the bug or if it was something else. If someone is also experiencing this issue maybe this will help them.

To summarize, everything now works as expected:
* When clicking on the Customize option in the context menu on the **primary monitor** I can adjust the settings of the **primary monitor**.
* When clicking on the Customize option in the context menu on the **secondary monitor** I can adjust the settings of the **secondary monitor**.
